dev-tools/mage/gotest.go

```diff
@@ -93,6 +95,32 @@ func makeGoTestArgsForModule(name, module string) GoTestArgs {
 	return params
 }
 
+// fetchGoPackages uses "go list ./module/..." to get all Go packages for a
```

Comment:
Could comment be clarified? It's a little confusing since all of this happens under a directory actually named "module", but what is actually run here is "go list ./<module name>/..."